examples/util.h

```diff
@@ -6,6 +6,18 @@
 
 namespace examples {
 
+#if defined(_WINDOWS)
+#define OTIO_EXAMPLES_MAIN() int wmain(int argc, wchar_t* argv[])
```

Comment:
@meshula Thanks for reviewing the changes. In the tlRender screenshot I opened the file via Qt, I was just checking whether the changes in deserialization.cpp/serialization.cpp were working OK.

I agree that the wmain changes are unusual, and probably not appropriate for the example code. But in my test code it looks like ```main()``` is not getting UTF8 on Windows, unless I'm doing something totally wrong which is certainly possible.

For example this code with ```main()``` returns "-1":
```
// Compare the char* command-line argument to a UTF8 string.
// Returns: -1
int main(int argc, char* argv[])
{
    return strcmp(argv[1], "大平原.otio");
}
```

While this code with ```wmain()``` returns "0":
```
// Convert the wchar_t* command-line argument to UTF8 and compare with a UTF8 string.
// Returns: 0
int wmain(int argc, wchar_t* argv[])
{
    const int bytes = WideCharToMultiByte(CP_UTF8, 0, argv[1], -1, NULL, 0, NULL, NULL);
    std::vector<char> chars(bytes);
    WideCharToMultiByte(CP_UTF8, 0, argv[1], -1, chars.data(), bytes, NULL, NULL);
    return strcmp(chars.data(), "大平原.otio");
}
```

And printing out the chars from ```main()```:
```
// Print the values of the char* command-line argument and UTF8 string.
int main(int argc, char* argv[])
{
    const int l = strlen(argv[1]);

    const char s[] = "大平原.otio";
    const int sl = strlen(s);
    
    for (int i = 0; i < std::max(l, sl); ++i)
    {
        printf("%3d %3d\n", i < l ? (argv[1][i]) : 0, i < sl ? (s[i]) : 0);
    }
}
```

Prints this out:
```
 63 -27
 63 -92
 63 -89
 46 -27
111 -71
116 -77
105 -27
111 -114
  0 -97
  0  46
  0 111
  0 116
  0 105
  0 111
```

The command-line argument was passed to the programs via the Visual Studio "Debugging/Command Arguments" property. The source file was converted to UTF-8 encoding with Notepad++.

Reply:
Hmm... The print out on the left side is the ASCII "???.otio". And if I try and run the test program by pasting the Unicode file name into a command prompt it also appears as "???.otio". If I make a .bat file in UTF-8 encoding to run the test program, I get this:
```
115 -27
-15 -92
-70 -89
115 -27
-90 -71
-90 -77
115 -27
-60 -114
-125 -97
 46  46
111 111
116 116
105 105
111 111
```
Bizarre...

Reply:
I wouldn't expect strcmp to work on a UTF8 string outside of the Latin-8 page, and there's an infinite number of reasons for the third program fragment to not work starting with the encoding of the source file itself, and whether the console has utf-8 as the active console codepage, even before you get to the main/wmain question :) C/C++, windows old consoles, and international strings are a match made in hell.

Reply:
Since the main/wmain question is all about the command line examples, and since this change doesn't currently address main/wmain, I think we are good to land this PR, and sort out the command line questions independent of addressing the really important fix you've provided here.